packages/uniforms/__tests__/AutoForm.tsx

```diff
@@ -27,66 +31,84 @@ describe('AutoForm', () => {
     validator.mockClear();
   });
 
-  describe('when changed', () => {
+  describe('when changes', () => {
     it('updates', () => {
-      // FIXME: AutoForm is not a valid Component.
-      const wrapper = mount<AutoForm | any>(
+      render(
         <AutoForm onChange={onChange} schema={schema} />,
+        { schema: { type: SimpleSchema2Bridge } },
+        { onChange },
       );
-
-      wrapper.instance().getContext().onChange('a', '2');
-
-      expect(onChange).toHaveBeenCalledTimes(1);
-      expect(onChange).toHaveBeenLastCalledWith('a', '2');
```

Comment:
This test has no assertions.

Reply:
fixed, but I need to verify why onChange is called 4 times. I think is unexpected behavior.

Reply:
I think we should consider changing <AutoFields> because now it adds fields iteratively which makes it execute n times onChangeModel, maybe we should collect them and then do createElement once ?